### ISSUE TITLE
Data Tables - global/master template

### ISSUE THREAD

**Issue by babalicious-io** [stampchain-io/BTCStampsExplorer#407]
The tables on the Stamp Detail and Token Detail need to be merged and a global/master template created.

**At the moment each page uses it's own structure and build.**

![Screenshot 2024-11-30 at 22 04 36](https://github.com/user-attachments/assets/2ca0a177-0eef-44d8-98b5-6b413866b825)

The Stamp Detail page is styled correctly ↑

![Screenshot 2024-11-30 at 22 04 59](https://github.com/user-attachments/assets/aae50c49-5327-4acb-b02a-e03061fa42a1)

The Token Detail page - Mints ↑
Transfers ↓

![Screenshot 2024-11-30 at 22 05 19](https://github.com/user-attachments/assets/b0a9a8e1-b354-4394-8cd9-1ef00fafedd9)

I suggest we use the Stamp Details page table as the foundation for a global/master template that the Token Details page also uses.

Files:
**Stamp Detail page**
- StampRelatedInfo - parent
- StampDispensers
- StampSales
- StampTransfers

**Token Detail page**
- SRC20DetailsTab
- SRC20TX

**Comment by reinamora137** [comment]
yeah perhaps something in constants or a shared template style page to keep a central source for these is good with me for now.

**Comment by babalicious-io** [comment]
should be solved with this update
https://github.com/stampchain-io/BTCStampsExplorer/pull/651
